clientconn.go

```diff
@@ -688,7 +700,6 @@ func (cc *ClientConn) switchBalancer(name string) {
 		grpclog.Infoln("ignoring service config balancer configuration: WithBalancer DialOption used instead")
 		return
 	}
-
 	if cc.curBalancerName == name {
```

Comment:
Should this go above the log about switching?

Reply:
Right, but we don't need to log that we aren't switching if the name doesn't change, right?

Reply:
Done